frontend/src/MuiForms/CustomTextInput.tsx

```diff
@@ -12,40 +14,48 @@ interface EntitySelectorProps {
   InputProps?: any
 }
 
-export default function EntitySelector(props: EntitySelectorProps) {
+export default function CustomTextInput(props: CustomTextInputProps) {
   const { onChange, value: currentValue, label, formContext } = props
 
-  const _onChange = (e) => {
-    onChange(e.target.value)
-  }
-
   const theme = useTheme()
 
+  const _onChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    onChange(event.target.value)
+  }
+
   return (
-    <TextField
-      {...props}
-      size='small'
-      sx={{
-        input: {
-          color: theme.palette.mode === 'light' ? 'black' : 'white',
-        },
-        label: {
-          WebkitTextFillColor: theme.palette.mode === 'light' ? 'black' : 'white',
-        },
-        '& .MuiInputBase-input.Mui-disabled': {
-          WebkitTextFillColor: theme.palette.mode === 'light' ? 'black' : 'white',
-        },
-      }}
-      onChange={_onChange}
-      variant={!formContext.editMode ? 'standard' : 'outlined'}
-      label={label}
-      required={!formContext.editMode ? false : true}
-      value={currentValue || ''}
-      disabled={!formContext.editMode}
-      InputProps={{
-        ...props.InputProps,
-        disableUnderline: !formContext.editMode ? true : false,
-      }}
-    />
+    <Fragment key={label}>
+      <Typography sx={{ fontWeight: 'bold' }}>{label}</Typography>
+      <TextField
+        size='small'
+        sx={{
+          input: {
+            color: theme.palette.mode === 'light' ? 'black' : 'white',
+          },
+          label: {
+            WebkitTextFillColor: theme.palette.mode === 'light' ? 'black' : 'white',
```

Comment:
I'm guessing that `WebkitTextFillColor` is setting the `-webkit-text-fill-color` CSS property. If that's the case then we should be able to just use `color` instead, unless MUI is setting `-webkit-text-fill-color` under the hood. 

The MDN docs on this property state:
"The -webkit-text-fill-color CSS property specifies the fill [color](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value) of characters of text. If this property is not set, the value of the [color](https://developer.mozilla.org/en-US/docs/Web/CSS/color) property is used"
See: https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-text-fill-color

Reply:
Using `color` does not work unfortunately